x-pack/plugins/infra/public/pages/metrics/layouts/container.ts

```diff
@@ -16,35 +17,47 @@ import {
 export const containerLayoutCreator: InfraMetricLayoutCreator = theme => [
   {
     id: 'containerOverview',
-    label: 'Container',
+    label: i18n.translate('xpack.infra.containerMetrics.containerLabel', {
+      defaultMessage: 'Container',
+    }),
     sections: [
       {
         id: InfraMetric.containerOverview,
-        label: 'Overview',
+        label: i18n.translate('xpack.infra.containerMetrics.overviewSectionLabel', {
+          defaultMessage: 'Overview',
+        }),
         requires: ['docker.cpu', 'docker.memory', 'docker.network'],
         type: InfraMetricLayoutSectionType.gauges,
         visConfig: {
           seriesOverrides: {
             cpu: {
-              name: 'CPU Usage',
+              name: i18n.translate('xpack.infra.containerMetrics.visConfig.cpuUsageName', {
+                defaultMessage: 'CPU Usage',
+              }),
               color: theme.eui.euiColorFullShade,
               formatter: InfraFormatterType.percent,
               gaugeMax: 1,
             },
             memory: {
-              name: 'Memory Usage',
+              name: i18n.translate('xpack.infra.containerMetrics.visConfig.memoryUsageName', {
```

Comment:
Accordingly, this would be the label of the "memory usage" time series in the "overview" section, so how about `xpack.infra.metricDetailPage.containerMetricsLayout.overviewSection.memoryUsageSeriesLabel`?